bin/webpack-entries.js

```diff
@@ -151,6 +151,8 @@ const entries = {
 		...getBlockEntries( 'frontend.{t,j}s{,x}' ),
 		'mini-cart-component':
 			'./assets/js/blocks/mini-cart/component-frontend.tsx',
+		'product-button-interactivity':
+			'./assets/js/atomic/blocks/product-elements/button/frontend.tsx',
```

Comment:
Do you know if this is required? I guess `getBlockEntries` already returns an entry for this block, am I right? 🤔

Reply:
`getBlockEntries` is configurated only for the blocks in the path: `assets/js/blocks`. If you check the implementation ([source code](https://github.com/woocommerce/woocommerce-blocks/blob/ef954c9dbc7aa3e6291ccdf01141c314a6e4e920/bin/webpack-entries.js/#L101-L105)), we can't use the `customDir` option as well.

Since that we will remove the `atomic` block (https://github.com/woocommerce/woocommerce-blocks/issues/6038), I keep this change as it is.